stable/dotnetcore/Makefile

```diff
@@ -3,22 +3,22 @@ deploy: get-dotnetcore get-dotnetcore-dependency
 test: get-dotnetcore-verify get-dotnetcore-dependency-verify
```

Comment:
just to ensure we don't break 2.0, could you add two cases `get-dotnetcore20` and `get-dotnetcore20-verify`? (using the previous runtime)

Reply:
Good point! Gonna fix :)